modin/backends/pandas/query_compiler.py

```diff
@@ -1925,7 +1925,7 @@ def back(self, n):
     # End Head/Tail/Front/Back
 
     # __getitem__ methods
-    def getitem_column_array(self, key):
+    def getitem_column_array(self, key, numeric=False):
```

Comment:
Can we add some documentation for the kwarg `numeric`?

Reply:
Yes, I'm not thrilled about adding the parameter, but getting columns and getting rows have different semantics and we need this to be able to transfer between them. I will document it.